extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts

```diff
@@ -186,6 +186,18 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
 					variableName: VariableNames.AppReaders_VariableName,
 					placeHolder: localize('deployCluster.AppReadersPlaceHolder', "Use comma to separate the values."),
 					description: localize('deployCluster.AppReadersDescription', "The Active Directory users or groups of app readers. Use comma as separator them if there are multiple users/groups.")
+				}, {
+					type: FieldType.Text,
+					label: localize('deployCluster.SubDomain', "Sub domain"),
+					required: false,
+					variableName: VariableNames.SubDomain_VariableName,
+					description: localize('deployCluster.SubDomainDescription', "A unique DNS subdomain to use for this BDC cluster. If not provided, cluster name will be used as the default value.")
+				}, {
+					type: FieldType.Text,
+					label: localize('deployCluster.AccountPrefix', "Account prefix"),
+					required: false,
+					variableName: VariableNames.AccountPrefix_VariableName,
+					description: localize('deployCluster.AccountPrefixDescription', "A unique prefix for AD accounts BDC cluster will generate,When not provided, subdomain name will be used as the default value. When subdomain is not provided, cluster name will be used as the subdomain name.")
```

Comment:
`BDC cluster` == `Big Data Clusters cluster`? Seems like normally we use the full product name `Big Data Cluster` when referring to an actual cluster